.github/workflows/fuzzers.yml

```diff
@@ -95,8 +95,8 @@ jobs:
           # - afl_qemu
           # - honggfuzz_qemu
           # - weizz_qemu
-          # - aflplusplus_qemu
-          # - aflplusplus_frida
+          - aflplusplus_qemu
+          - aflplusplus_frida
```

Comment:
You don't need to do this anymore.

Reply:
Not? I get a CI error 3 weeks ago where I forgot that if I remember correctly